Pipfile

```diff
@@ -10,6 +10,8 @@ gunicorn = "*"
 gevent = "*"
 sdc-cryptography = "*"
 google-cloud-pubsub = "*"
+setuptools = "*"
```

Comment:
idna and setuptools are subpackages of google-cloud-pubsub, don't think we want to explicitly declare them in the pipfile.